packages/type-safe-api/docs/developer_guides/type-safe-api/websocket_using_openapi.md

```diff
@@ -0,0 +1,99 @@
+# Using OpenAPI
+
+As an alternative to [Smithy](./using_smithy.md), you can use [OpenAPI Version 3.0.3](https://swagger.io/specification/) to define your API.
+
+!!!warning
+
+    Note that while Type Safe WebSocket API uses the OpenAPI format to define operations, many of the features of OpenAPI are not supported by WebSockets and therefore may be ignored if you define them in your model, for example specifying `parameters` or `responses`.
```

Comment:
We should probably call out explicitly that Smithy is the recommended model lang.